src/room.ts

```diff
@@ -190,6 +207,8 @@ export class Room extends EventEmitter implements Sayable {
 
     const memberList  = this.parseMemberList(rawObj.MemberList)
     const nickMap     = this.parseNickMap(rawObj.MemberList)
+    const remarkMap   = this.parseRemarkMap(rawObj.MemberList)
+    const displayMap  = this.parseDisplayName(rawObj.MemberList)
```

Comment:
Instead of introduce new functions, use `this.parseMap(memberList, 'display')` directly at here.